### PR TITLE
Fixed voting from poll view page

### LINKED ISSUE #847: Cannot vote from Poll View page
When clicking 'Vote' on a vote view page (e.g. [for the current poll](https://gewis.nl/poll/484/view), general form _/poll/[pollId]/view_) it will redirect the user to another page on which (I assume) the vote from the user is handled. However this page ([for the current poll](https://gewis.nl/poll/484/vote), general form _/poll/[pollId]/vote_) throws a 500 Internal Server error.

I think the page expects a form value from the vote submission, but it does not exist[1] since on the vote view page there is not an option to select a value for which one can vote.

Just clicking on 'Vote' from the front page results in the same error (sort of confirms [1]).

I cannot check what exactly causes the error because my local copy of the website is not cooperating, so I think someone should check the logs.

If this is not how I am supposed to let you know about issues like this, please let me know.

### PR DESCRIPTION
**Pull request by Koen1999**
This resolves #847 

The cause was just the lack of radio buttons, or in other words, missing input for the post request.

### CODE REVIEW

File: module/Frontpage/view/partial/poll.phtml [hunk 1]
```diff
@@ -27,17 +27,31 @@ $lang = $this->plugin('translate')->getTranslator()->getLocale();
...
 
-        <?php if ($showResults || !$canVote): ?>
-            <div>
-                <h5 class="clearfix<?= $voted ? ' text-primary' : '' ?>">
-                    <?= $text ?>
-                    <small class="pull-right" style="margin-top: 0.4em;"><?= $option->getVotesCount() === 1 ? $this->translate('1 vote') : sprintf($this->translate('%d votes'), $option->getVotesCount()) ?></small>
-                </h5>
+        <?php if ($showResults && $canVote): ?>
+            <div class="radio">
+                <label>
```
Comment (tomudding) [inline module/Frontpage/view/partial/poll.phtml]: Needs a `width: 100%` attribute (either inline or applied to the `.radio label` selector) to fix the compressed view.

File: module/Frontpage/view/partial/poll.phtml [hunk 2]
```diff
@@ -27,17 +27,31 @@ $lang = $this->plugin('translate')->getTranslator()->getLocale();
...
+        <div>
+            <h5 class="clearfix<?= $voted ? ' text-primary' : '' ?>">
+                <?= $text ?>
+                <small class="pull-right" style="margin-top: 0.4em;"><?= $option->getVotesCount() === 1 ? $this->translate('1 vote') : sprintf($this->translate('%d votes'), $option->getVotesCount()) ?></small>
+            </h5>
+
+            <div class="progress progress--poll">
+                <div class="progress-bar" style="width: <?= $percentages[$option->getId()] ?>%;"></div>
             </div>
+        </div>
```
Comment (tomudding) [inline module/Frontpage/view/partial/poll.phtml]: Missing indentation in comparison to all other code.

File: module/Frontpage/view/partial/poll.phtml [hunk 3]
```diff
@@ -27,17 +27,31 @@ $lang = $this->plugin('translate')->getTranslator()->getLocale();
...
-        <?php if ($showResults || !$canVote): ?>
-            <div>
-                <h5 class="clearfix<?= $voted ? ' text-primary' : '' ?>">
-                    <?= $text ?>
-                    <small class="pull-right" style="margin-top: 0.4em;"><?= $option->getVotesCount() === 1 ? $this->translate('1 vote') : sprintf($this->translate('%d votes'), $option->getVotesCount()) ?></small>
-                </h5>
+        <?php if ($showResults && $canVote): ?>
+            <div class="radio">
+                <label>
+                    <input type="radio" value="<?= $option->getId() ?>" name="option">
```
Comment (tomudding) [inline module/Frontpage/view/partial/poll.phtml]: QoL Improvement: centre radio button vertically.

**Comment by Koen1999** [comment]
I have improved the layout, it now looks as follows:

afbeelding

File: module/Frontpage/view/partial/poll.phtml (hunk 1 above)
Comment (Koen1999) [reply module/Frontpage/view/partial/poll.phtml]: I resolved this in another way

File: module/Frontpage/view/partial/poll.phtml (hunk 2 above)
Comment (Koen1999) [reply module/Frontpage/view/partial/poll.phtml]: Solved this

File: module/Frontpage/view/partial/poll.phtml (hunk 3 above)
Comment (Koen1999) [reply module/Frontpage/view/partial/poll.phtml]: This is something for another time I guess

**Comment by Koen1999** [comment]
The PR was checked during the working moment.

**Pull request merged**